_data/roadmap-items.yml

```diff
@@ -9,6 +9,7 @@ fundamentals:
   text: You need a foundational grasp of Computer Science concepts in order to think your way around code. You don't necessarily need this in order to build with code... but you do need it in order to understand what you're building and debugging.
   links:
     links: Check out <a href="/comp-sci/2016/10/01/general-uncategorized/">our list of fundamental computer science concepts</a>.
+    roadmap: Take a look at <a href="https://github.com/kamranahmedse/developer-roadmap/"> this flow chart</a> of Computer Science topics. It may seem like a lot, but don't worry! You won't need to know everything right away.This a useful reference if you're wondering where a topic fits in.
```

Comment:
Also add an attribute `target='_blank'` to help keep users on the page while they click around!